signup-default.properties

```diff
@@ -0,0 +1,93 @@
+## Application Name
+spring.application.name=signup
+
+server.port=8088
+server.servlet.path=/
+spring.mvc.servlet.path=${server.servlet.path}
+server.servlet.context-path=/v1/signup
+
+openapi.info.title=Signup Service
+openapi.info.description=Rest Endpoints for operations related to signup service
+openapi.info.version=1.0
+openapi.info.license.name=Mosip
+openapi.info.license.url=https://docs.mosip.io/platform/license
+mosipbox.public.url=http://localhost:${server.port}
+openapi.service.server.url=${mosipbox.public.url}${server.servlet.context-path}
+openapi.service.server.description=SignUp Service
+springdoc.swagger-ui.disable-swagger-default-url=true
+
+spring.messages.basename=messages
+spring.messages.encoding=UTF-8
+
+spring.main.allow-bean-definition-overriding=true
```

Comment:
From line 1 to line 22 should not be added here, it will stay in bootstrap.properties of signup-service